### PR TITLE
fix missing core dependency

### PR DESCRIPTION
**Pull request by pelikhan**
Fixes compilation via the CLI

### CODE REVIEW

**Comment by pelikhan** [comment]
@emwta

**Comment by pelikhan** [comment]
@microbit-mark FYI

File: pxt.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
     "version": "1.0.15",
     "description": "A iBit package for pxt-microbit",
     "license": "MIT",
-    "dependencies": {},
+    "dependencies": {
```
Comment (emwta) [inline pxt.json:6]: [run] git status --porcelain --untracked-files=no
ERROR: Package not installed: core

How fix it?

Comment (pelikhan) [reply pxt.json:6]: run ```pxt install```

**Comment by emwta** [comment]
fix it, done.

**Comment by pelikhan** [comment]
Don’t forget to bump

**Comment by emwta** [comment]
I ``bump`` finished.

**Pull request merged**